### ISSUE TITLE
CSS `legacyValue` is no longer needed

### ISSUE THREAD

**Issue by cdoublev** [w3c/reffy#1215]
This [commit](https://github.com/w3c/csswg-drafts/commit/3e7df1c6cbfa1f97b2b323bc2dc6a30273ab0f5e) in CSS Color 4 explicitly defines legacy syntaxes for `rgb()`, `rgba()`, `hsl()`, `hsla()`, which means they do not require to be [defined with `legacyValue`](https://github.com/w3c/reffy/blob/63342758be83848e1446fda72418b06b3481783f/src/browserlib/extract-cssdfn.mjs#L531) anymore.

However the online specs do not seem to reflect these changes yet, and I expressed [my concern](https://github.com/w3c/csswg-drafts/issues/8231#issuecomment-1438583742) about another change in CSS Color 5 that prevents extracting legacy and modern syntaxes from CSS Color 4 + the "advanced" modern syntax from CSS Color 5 (with relative color keyword and missing components in color functions).

**Comment by tidoust** [comment]
Thanks for the heads-up. The logic in Reffy is still needed for the [/TR/css-color-4/](https://www.w3.org/TR/css-color-4/#rgb-functions) spec for now (which may be what you call "online specs"?). At least in theory... In practice, looking into the code and history of extracts more thoroughly, I note that logic actually broke in [November last year](https://github.com/w3c/webref/commit/15f0c80b0de078d46123cce51b9e2791a73ea5bd#diff-ba25acce8c1cdbc4f3aab84575941a1c8c8c6007d6860729fcb5285deacedb1b) when we dropped surrounding `<>` for functions because the code is still looking for `<rgb()>` and not for `rgb()`.

I'll use this opportunity to drop the custom logic entirely.

Thanks for raising the issue against css-color-5.

**Comment by cdoublev** [comment]
I was referring to drafts.csswg.org, as opposed to the Github repository. 

Thanks for the quick fix. I guess we are now waiting for the fix in CSS Color 5 before we can close this issue.

**Comment by tidoust** [comment]
I was more waiting for PR #1218 to be merged and released. Now done. That should be all that's needed from Reffy's perspective.
I created an issue on Webref to track the possible creation of a patch to add the legacy syntax if it does not get re-introduced in CSS Color 5.

**Comment by cdoublev** [comment]
Yeah, I was wrong saying that Reffy would ignore legacy syntaxes from CSS Color 4. It will only [affect](https://github.com/w3c/webref/blob/60ebf71b9d555c523975cfefb08f5420d12b7293/tools/drop-css-property-duplicates.js#L177) the Webref NPM package, if there is a release before. Thanks again. 👍

**Comment by svgeesus** [comment]
>  I guess we are now waiting for the fix in CSS Color 5 before we can close this issue.

I just did those fixes
